firefox-ios/Client/Configuration/version.xcconfig

```diff
@@ -1 +1 @@
-APP_VERSION = 138.0
+APP_VERSION = 139.0
```

Comment:
It seems this should be a correct change since we are in `139.0`, but do we know why we're adding this change here?